test/src/integration/integration_scalardl_test.go

```diff
@@ -115,3 +121,25 @@ func getExternalIP(t *testing.T) string {
 
 	return output
 }
+
+func isReachable(t *testing.T, host string) bool {
+	logger.Logf(t, "Check tcp connection: %s", host)
+
+	numRetries := 60
```

Comment:
You really mean to retry 60 times?
(or you want to retry 6 times in 60 seconds?)

Reply:
Yes, I want to retry 60 times with 10 second intervals.